src/openforms/js/initTinymce.js

```diff
@@ -34,12 +34,12 @@
 
       // TODO Django 4.2: use explicit theme names rather than the media query approach:
       // https://github.com/django/django/blob/main/django/contrib/admin/static/admin/css/dark_mode.css#L36
-      const useDarkMode = window.matchMedia('(prefers-color-scheme: dark)').matches;
+      let dark = 'dark';
```

Comment:
we can only make this change when open-forms itself updates to Django 4.2! For now, we're still on 3.2

Reply:
@ErhanCitil can you please revert this change?